cfme/tests/infrastructure/test_timelines.py

```diff
@@ -16,21 +20,21 @@
 excluded = ProviderFilter(classes=[SCVMMProvider, RHEVMProvider], inverted=True)
 pytestmark = [
     pytest.mark.tier(2),
-    pytest.mark.provider(gen_func=providers, filters=[excluded, all_infra_prov], scope='module'),
-    pytest.mark.usefixtures('setup_provider_modscope')
+    pytest.mark.provider(gen_func=providers, filters=[excluded, all_infra_prov]),
+    pytest.mark.usefixtures('setup_provider')
 ]
 
 
-@pytest.fixture(scope='module')
-def new_vm(request, provider):
+@pytest.fixture(scope='function')
```

Comment:
you don't need to set scope to function. it is default one

Reply:
ok. and on the override, should I remove it as well? 
@pytest.mark.provider([VMwareProvider], override=True, scope='function')